webservices/server-integration/src/main/java/org/jboss/as/webservices/logging/WSLogger.java

```diff
@@ -128,7 +128,7 @@
     @Message(id = 17, value = "Invalid handler chain file: %s")
     void invalidHandlerChainFile(String fileName);
 
-    String WS_SPEC_REF_5_3_2_4_2 = ". See section 5.3.2.4.2 of \"Web Services for Java EE, Version 1.4\".";
+    String WS_SPEC_REF_5_3_2_4_2 = ". See section 5.3.2.4.2 of \"Jakarta Enterprise Web Services 1.4\".";
```

Comment:
@asoldano Technically, there is no such thing:

https://jakarta.ee/specifications/enterprise-ws/1.4/enterprise-ws-spec-1.4.html

There's Section 1 and that's it.

Reply:
It looks like the invalid asciidoc issues were fixed for EE 9 via https://jakarta.ee/specifications/enterprise-ws/2.0/enterprise-ws-spec-2.0.html.